### PR TITLE
Realise UpdateBullet function

### PR DESCRIPTION
**Pull request by haritonn**
add a function that determines the state of the bullet

### CODE REVIEW

File: user.cpp [hunk 1]
```diff
@@ -341,8 +341,11 @@ void ShootBullet(Context &ctx, Object &player, float dt)
 //
 void UpdateBullet(Context &ctx, Object &obj, float dt)
 {
+    obj.position += obj.bullet.speed;
```
Comment (vasthecat) [inline user.cpp:344]: Speed should be multiplied by `dt`, so that it doesn't depend on FPS of application.

**Review by vasthecat — CHANGES_REQUESTED** [review]

**Comment by haritonn** [comment]
I actually have no idea why I merged the remote branch with the local, it was like testing idk :/

**Review by mchernigin — APPROVED** [review]
Кажется, всё работает корректно

**Review by vasthecat — APPROVED** [review]
LGTM

**Pull request merged**